manager/api/export-import/src/main/java/io/apiman/manager/api/exportimport/manager/ExportImportManager.java

```diff
@@ -36,10 +36,8 @@
     
     @Inject
     private ExportImportConfigParser config;
-    @Inject @ApimanLogger(IImportReader.class)
-    private IApimanLogger importLogger;
-    @Inject @ApimanLogger(IExportWriter.class)
-    private IApimanLogger exportLogger;
+    private IApimanLogger importLogger = ApimanLoggerFactory.getLogger(IImportReader.class);
```

Comment:
Any reason to not make this also `static final`?

Reply:
Refer to the previous comment. 

I did this initially and had to back it out. It turns out that the implementation of ImportExport uses the logger to collect some logging information that is put into the HTTP response body when you do an import/export via https://github.com/msavy/apiman/blob/add-jboss-logging-to-wf-eap-distros/manager/api/rest-impl/src/main/java/io/apiman/manager/api/rest/impl/SystemResourceImpl.java#L177-L244

By changing this to `private final static` this trick doesn't work properly any more. 

It's an unusual use of a logger, but I want to just maintain compatibility for now.